pkgs/tools/system/auto-epp-go/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, lib, fetchurl, autoPatchelfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "auto-epp-go";
+  version = "0.1.1-1";
+
+  src = fetchurl {
+    url = "https://github.com/tfkhdyt/auto-epp-go/releases/download/v${version}/auto-epp-go-v${version}-linux-amd64.tar.gz";
```

Comment:
it's preferred to build from source, rather than package binaries

Reply:
Updated